public/content.css

```diff
@@ -1,7 +1,3 @@
-html {
-    color-scheme: dark;
```

Comment:
Was there a reason for this one?

Reply:
Does this file change the styles for the elements that are embedded in the YT player? If so, then, indeed, this rule does not seem to make sense, since I have not seen a scroll bar anywhere in them.